src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
-}
+
+  let count = 0;
+  const getSum = (valueFirst) => {
+    const getSumSecond = (valueSecond) => {
+      count++;
+
+      return count > 3 && count % 2 === 0
+        ? `Bzzz... Error!`
+        : valueFirst + valueSecond;
```

Comment:
It's a bad practice to use ternary operators with double conditions.
Two ways to fix it:
1. Move condition check to variable, smth like `shouldReturnError`
2. Rewrite whole structure using `if`

First one is more elegant.